htmresearch/frameworks/nlp/classify_keywords.py

```diff
@@ -19,45 +19,30 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-import copy
 import numpy
-import os
-
-import simplejson as json
+import random
 
 from htmresearch.frameworks.nlp.classification_model import ClassificationModel
 from nupic.algorithms.KNNClassifier import KNNClassifier
 
 
-
 class ClassificationModelKeywords(ClassificationModel):
   """
-  Class to run the survey response classification task with random SDRs.
-
-  From the experiment runner, the methods expect to be fed one sample at a time.
+  Class to run NLP classification task with random SDRs.
   """
-  # TODO: use nupic.bindings.math import Random
+  # TODO: use nupic.bindings.math import Random?
```

Comment:
I'd rather not, but this was requested by Scott a while back (I think for serialization concerns).